src/Admin/Filter/FilterValidationObject.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace Arbory\Base\Admin\Filter;
+
+class FilterValidationObject
+{
+
+    /**
+     * @var array
+     */
+    protected $rules;
+
+    /**
+     * @var array
+     */
+    protected $messages;
+
+    /**
+     * @var array
+     */
+    protected $attributes;
+
+    /**
+     * @var array
+     */
+    protected $transformers;
+
+    /**
+     * FilterValidationObject constructor.
+     * @param array $data
+     */
+    public function __construct(array $data = [])
+    {
+        $this->rules = $data['rules'] ?? [];
+        $this->messages = $data['messages'] ?? [];
+        $this->attributes = $data['attributes'] ?? [];
+        $this->transformers = $data['transformers'] ?? [];
+    }
+
+    /**
+     * @param array $rule
+     */
+    public function addRule(array $rule)
```

Comment:
Name is not accurate since it receives an array of rules, should also return `$this`
Same thing applies to `addMessage`, `addAttribute`